spec/lib/dfe/reference_data/qualifications_spec.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.describe DfE::ReferenceData::Qualifications::QUALIFICATIONS do
+  describe 'qualifications' do
+    it 'is a valid autocomplete-capable list' do
+      DfE::ReferenceData::Qualifications::QUALIFICATIONS.validate_autocomplete_compatibility!
+    end
```

Comment:
By the way, does the .validate_autocomplete_compatibility! method is used only on specs?

I think we could do an RSpec Matcher. Something like:

```ruby
RSpec::Matchers.define :be_valid_autocomplete_compatible do
  match do |actual|
    # actual is the described classes
    actual.all.each do |record|
      # do all the validate code on reference list
    end
  end
end
```

Unless you have plans to use in the implementation, I'd think a matcher would fit nicely. It is not a blocker though.

Reply:
I tried creating an RSpec matcher in my draft PR and just got lost in a confusing error message - I shall copy your example here and see if I can make this work! What would be a good fiole to put "shared" rspec logic like the matcher and shared example in? I presume not just hidden in one of the `name-of-thing_spec.rb` files...